matching-notification-ui/src/main/resources/MatchingNotification/PatientMatchNotificationEmailTemplate.xml

```diff
@@ -98,6 +98,9 @@
 #set($space-between-matches = 'margin-top : 25px;')
 #set($disclaimer = 'font-size : 10px;')
 #set($patient-id = 'white-space : no-wrap;')
+#if ("$!subjectPatient.serverId" != "")
```

Comment:
Shouldn't this be `#if (!`, meaning that the patient shouldn't have a server ID, since it's supposed to be local?